src/_pages/prototyping_boards/electronic-use.md

```diff
@@ -68,7 +68,39 @@ We created an open-source **Python library** managing this JSON API called [*Pyl
 
 Get <a href="https://github.com/Luos-io/Pyluos" target="_blank">pyluos on github</a>.
 
-## Update board's firmware
+## Update Luos, Robus and board's firmware
+
+To Update Luos, Robus and module's code you must go on the librairies window of platformio in Visual Studio Code:
+        <img src="{{img_path}}/Update_Luos_Robus_Modules.png" height="" />
+
+
+To update the board's firmware you need to follow 3 steps:
+
+    1. Open the module folder that you want to update in Visual Studio Code
+            To do this, you must go to "file" -> "open the folder" and go to the folder of the module you want
```

Comment:
I think, the user should go on the PIO tab (on the left) before and click "Open project" in PIO Home